modules/apps/29-fee/keeper/escrow.go

```diff
@@ -44,66 +44,143 @@ func (k Keeper) EscrowPacketFee(ctx sdk.Context, identifiedFee *types.Identified
 }
 
 // DistributeFee pays the acknowledgement fee & receive fee for a given packetId while refunding the timeout fee to the refund account associated with the Fee
-func (k Keeper) DistributeFee(ctx sdk.Context, refundAcc, forwardRelayer, reverseRelayer sdk.AccAddress, packetID *channeltypes.PacketId) error {
+func (k Keeper) DistributeFee(ctx sdk.Context, refundAcc, forwardRelayer, reverseRelayer string, packetID *channeltypes.PacketId) error {
+	var packetIdErr error
+	var distributeFwdErr error
+	var distributeReverseErr error
+	var refundAccErr error
```

Comment:
```suggestion
	var (
	    packetIdErr error
	    distributeFwdErr error
	    distributeReverseErr error
	    refundAccErr error
)
```

group vars (commit changes locally, this isn't formatted)